SurveyApp/Modules/Splash/SplashModule.swift

```diff
@@ -35,6 +35,10 @@ final class SplashModule {
     init() {
         let userSessionProvider = UserSessionProvider.shared
         
+        if ProcessInfo.processInfo.environment[TestConstants.clearUserSession] != nil {
+            userSessionProvider.clearCredentials()
+        }
+        
```

Comment:
I think it's better to create a class like this 
```

final class UserSessionStorageMock: UserSessionStorageProtocol {

    var storedSessionId: String?
    var storedUserType: String?
    var storedBiometricRefreshToken: String?

    var sessionId: String? {
        get { storedSessionId }
        set { storedSessionId = newValue }
    }

    var userType: LoginUserType? {
        get { LoginUserType(rawValue: storedUserType.string) }
        set { storedUserType = newValue?.rawValue }
    }

    var biometricRefreshToken: String? {
        get { storedBiometricRefreshToken }
        set { storedBiometricRefreshToken = newValue }
    }

    func clear() {
        storedSessionId = nil
        storedUserType = nil
        storedBiometricRefreshToken = nil
    }
}
```
and use it instead of the real storage.
```

        let userStorage: UserSessionStorageProtocol = when(
            UITestHelper.shared.shouldUseMockStorage,
            UserSessionStorageMock(),
            UserSessionStorage.shared
        )
        NetworkServiceFactory.shared.setUp(
            baseURL: Constants.Network.baseURLPath,
            sessionStorage: userStorage
        )
```